### ISSUE TITLE
Is there a full version of the code that implements tsm?Contains input and loss

### ISSUE THREAD

**Issue by l2009312042** [Pika7ma/Temporal-Shift-Module#2]

**Comment by PingchuanMa** [comment]
I provided a demo with backbone of resnet in the repo. You can simply use it as a utility module using the code from TSN-PyTorch.

**Comment by wangtz** [comment]
Thanks for working on this. I noticed that the model was pre-trained on Kinetics dataset, however TSN-PyTorch (https://github.com/yjxiong/tsn-pytorch) only provides utilities for UCF 101. Would you recommend what's the best approach to generate frame list for Kinetics? Thanks!
